msteams-platform/concepts/resource-specific-consent.md

```diff
@@ -0,0 +1,117 @@
+
+#### 4. Get an access token 
+
+Before you make a REST call to the Graph, you need to [get an access token](/graph/auth-v2-service) for the application permissions similar to getting an application permission token for non-RSC use.
+You specify the pre-configured permissions by passing `https://graph.microsoft.com/.default` as the value for the `scope` parameter in the token request. See the `scope` parameter description in the token request below for details.
+
+### Token request
+
+```
+POST https://login.microsoftonline.com/{tenant}/oauth2/v2.0/token
+Host: login.microsoftonline.com
+Content-Type: application/x-www-form-urlencoded
+
+grant_type=client_credentials
+&client_id={appId}
+&client_secret={appSecret}
+&scope=https%3A%2F%2Fgraph.microsoft.com%2F.default
+
+```
+
+| Parameter     | Condition | Description 
+|:--------------|:----------|:------------
+| tenant        | Required | The directory tenant that you want to request permission from. This can be in GUID or a friendly name format.
+| client_id     | Required | The Application ID that the [Azure app registration portal](https://go.microsoft.com/fwlink/?linkid=2083908) assigned when you registered your app.
+| scope         | Required | The value passed for the `scope` parameter in this request should be the resource identifier (Application ID URI) of the resource you want, affixed with the `.default` suffix. For the Microsoft Graph, the value is `https://graph.microsoft.com/.default`. This value informs the Microsoft identity platform endpoint that of all the application permissions you have configured for your app, it should issue a token for the ones associated with the resource you want to use.
+| client_secret | Required | The Application Secret that you generated for your app in the app registration portal.
+| grant_type    | Required | Must be `client credentials`.
+
+#### Token response
+
+A successful response looks like this:
+
+```JSON
+{
+  "token_type": "Bearer",
+  "expires_in": 3599,
+  "access_token": "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsIng1dCI6Ik1uQ19WWmNBVGZNNXBP..."
+}
+```
+
+| Parameter     | Description
+|:--------------|:------------
+| access token | The requested access token. Your app can use this token in calls to Microsoft Graph.
+| token type    | Indicates the token type value. The only type that Azure AD supports is `bearer`.
+| expires in   | How long the access token is valid (in seconds).
+
+  
+#### 5. Make a Graph call 
+
+You can now make a REST call like a non-RSC call or use the SDK. See [Microsoft Graph](https://developer.microsoft.com/en-us/graph) and [Use the Microsoft Graph API to work with Microsoft Teams](/graph/api/resources/teams-api-overview?view=graph-rest-beta) for more details. 
```

Comment:
Huh?

Reply:
Can we not use methods available in [beta SDK](https://docs.microsoft.com/en-us/graph/api/channel-list-messages?view=graph-rest-beta&tabs=csharp#example) to call the APIs once we acquire the access token?

Reply:
Yes you can, but the first sentence is still very confusing. "make a REST call like a non-RSC call"?